### PR TITLE
Feat/Provide common params with `.yml` file

### PR DESCRIPTION
**Pull request by carpawell**

### CODE REVIEW

File: services/storage/docker-compose.yml [hunk 1]
```diff
@@ -23,8 +23,10 @@ services:
       - ../nats/client-cert.pem:/etc/neofs-node/nats.tls.cert
       - ../nats/client-key.pem:/etc/neofs-node/nats.tls.key
       - ../nats/ca-cert.pem:/etc/neofs-node/nats.ca.crt
+      - ./cfg:/etc/neofs-node/cfg
     stop_signal: SIGKILL
-    env_file: [ ".env", ".storage.env", ".int_test.env" ]
```
Comment (realloc) [inline services/storage/docker-compose.yml:27]: Maybe we should keep `.storage.env` to simplify overriding?

Comment (carpawell) [reply services/storage/docker-compose.yml:27]: does your suggestion include keeping the empty `.storage.env` in the repo?

File: services/ir/.ir.env [hunk 2]
```diff
@@ -1,44 +1,4 @@
...
-NEOFS_IR_CONTRACTS_PROCESSING=37d3232083dd2c1bed76bb28d1c992fc6262f699
-
-NEOFS_IR_MORPH_VALIDATORS=02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2
-NEOFS_IR_AUDIT_PDP_MAX_SLEEP_INTERVAL=100ms
-
-NEOFS_IR_SETTLEMENT_BASIC_INCOME_RATE=100000000
-NEOFS_IR_SETTLEMENT_AUDIT_FEE=100000
-
-NEOFS_IR_LOCODE_DB_PATH=/locode/db
+NEOFS_IR_CONTRACTS_PROCESSING=332016219b9eb48348fb12694409dbefe29bdf30
```
Comment (KirillovDenis) [inline services/ir/.ir.env:2]: Why we save some parameters in `.env`?

Comment (carpawell) [reply services/ir/.ir.env:2]: it is sad but some of our envs are [used](https://github.com/nspcc-dev/neofs-dev-env/blob/d8e825443ee73cc5fda38f83f11e8443a111ea1b/bin/deposit.sh#L21) in other places so simple removal breaks other places. i could not come up with a simple solution for that so decided to keep it as is and solve that in another PR (it needed)

Comment (KirillovDenis) [reply services/ir/.ir.env:2]: Then maybe we should not duplicate this parameters in [.yaml](https://github.com/carpawell/neofs-dev-env/blob/5841fed24ebcb583a5dc9a4f3d4814d38d1fe5f7/services/ir/cfg/config.yml#L66)?

Comment (carpawell) [reply services/ir/.ir.env:2]: oh, yeah, that's a mistake, fixed

**Comment by KirillovDenis** [comment]
What about `rest_gw`? Should we also create `config.yaml` for it?

**Comment by carpawell** [comment]
@KirillovDenis, yes, we can but it does not contain any `.rest.env` so i thought that it was an explicit decision. FYI: `docker-compose` for the `rest-gw` contains wallet only and _not used_ (as i understand) `REST_GW_LISTEN_ADDRESS`.

File: services/storage/docker-compose.yml [hunk 3]
```diff
@@ -23,8 +23,10 @@ services:
       - ../nats/client-cert.pem:/etc/neofs-node/nats.tls.cert
       - ../nats/client-key.pem:/etc/neofs-node/nats.tls.key
       - ../nats/ca-cert.pem:/etc/neofs-node/nats.ca.crt
+      - ./cfg:/etc/neofs-node/cfg
```
Comment (fyrchik) [inline services/storage/docker-compose.yml:26]: `/etc/neofs/storage`?

File: services/s3_gate/docker-compose.yml [hunk 4]
```diff
@@ -19,11 +19,11 @@ services:
       - ./tls.key:/tls.key
       - ./tls.crt:/tls.crt
       - ./../../vendor/hosts:/etc/hosts
+      - ./cfg:/etc/neofs-s3-gw/cfg
```
Comment (fyrchik) [inline services/s3_gate/docker-compose.yml:22]: Same, `/etc/neofs/s3`?

File: services/storage/docker-compose.yml (hunk 3 above)
Comment (carpawell) [reply services/storage/docker-compose.yml]: changed for all services

Comment (carpawell) [reply services/storage/docker-compose.yml]: changed for all services

**Comment by carpawell** [comment]
@KirillovDenis, added for `rest` too

**Review by KirillovDenis — APPROVED** [review]
LGTM

File: services/s3_gate/cfg/config.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+logger:
+  level: debug
+  
+# Timeout to check node health during rebalance
+healthcheck_timeout: 15s
```
Comment (masterSplinter01) [inline services/s3_gate/cfg/config.yml:5]: Is `rebalance_interval` dropped intentionally?

File: services/storage/cfg/config.yml [hunk 6]
```diff
@@ -0,0 +1,81 @@
+# Logger section
+logger:
+  level: debug # Minimum enabled logging level
+
+# Profiler section
+pprof:
+  enabled: true
+  address: :6060  # Server address
+  shutdown_timeout: 15s  # Timeout to shut down the HTTP server
```
Comment (masterSplinter01) [inline services/storage/cfg/config.yml:9]: `Timeout to shut down the HTTP server` sounds confusing a bit, maybe replace it with `# timeout for profiling HTTP server graceful shutdown`?

File: services/storage/cfg/config.yml [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+# Application metrics section
+prometheus:
+  enabled: true
+  address: :9090  # Server address
+  shutdown_timeout: 15s  # Timeout to shut down the HTTP server
```
Comment (masterSplinter01) [inline services/storage/cfg/config.yml:15]: `# timeout for metrics HTTP server graceful shutdown` ?

File: services/storage/cfg/config.yml (hunk 6 above)
Comment (carpawell) [reply services/storage/cfg/config.yml:9]: almost every comment was taken from the service's config examples OR from the previous `.env` config file

synced SN and IR with your suggestion

File: services/s3_gate/cfg/config.yml (hunk 5 above)
Comment (carpawell) [reply services/s3_gate/cfg/config.yml:8]: no, added back

**Review by masterSplinter01 — APPROVED** [review]

**Comment by roman-khimov** [comment]
@cthulhu-rider: how about merging this?

**Comment by notimetoname** [comment]
@roman-khimov, @cthulhu-rider nice PR, awesome [comment](https://github.com/nspcc-dev/neofs-dev-env/pull/241#issuecomment-1485403209).

**Comment by roman-khimov** [comment]
Maybe it needs some adjustments now though with updated dev-env? Could you check it still works fine?

**Comment by roman-khimov** [comment]
Poor @carpawell, he has made so many great things in NeoFS...

**Pull request closed without merging**